orderapi/src/main/java/fi/hel/verkkokauppa/order/service/order/OrderService.java

```diff
@@ -217,4 +231,38 @@ public void linkToSubscription(String orderId, String userId, String subscriptio
         order.setSubscriptionId(subscriptionId);
         orderRepository.save(order);
     }
+
+    public void triggerOrderCreatedEvent(Order order, String eventType) {
+        OrderMessage.OrderMessageBuilder orderMessageBuilder = OrderMessage.builder()
+                .eventType(eventType)
+                .namespace(order.getNamespace())
+                .orderId(order.getOrderId())
+                .timestamp(order.getCreatedAt())
```

Comment:
Should this be current timestamp? or should timestamp to be renamed to createdAt

Reply:
This because we want to have event trigger timestamp in every event.

Reply:
.timestamp(DateTimeUtil.getDateTime())
Can be used